packages/components/package.json

```diff
@@ -78,7 +78,7 @@
     "lodash.throttle": "^4.1.1",
     "prop-types": "^15.7.2",
     "react-content-loader": "^4.3.2",
-    "react-div-100vh": "^0.3.8",
+    "react-div-100vh": "^0.7.0",
```

Comment:
updated the package to support TS

Reply:
I guess we need to update `package-lock.json` too, not sure!

Reply:
@iman-fs quick question: Are you sure that this package update is within the scope of the current TS migration task? 😊

Reply:
@maryia-binary first I tried to fix the issue by adding https://www.npmjs.com/package/@types/react-div-100vh but it didn't work and as you can see it is deprecated. So there were either adding a decleration file (https://github.com/mvasin/react-div-100vh/issues/16 ) or updating the package and refactor the usage of it.

Reply:
@niloo-fs you're right. thanks